sdk/src/main/java/com/bugsnag/android/Stacktrace.java

```diff
@@ -9,6 +9,9 @@
  * where appropriate.
  */
 class Stacktrace implements JsonStream.Streamable {
+
+    private static final int STACKTRACE_TRIM_LENGTH = 200;
```

Comment:
200 seems reasonable. Have you done any size analysis to see what the typical payload size an error with 200 stackframes would be? If it's consistently under 1MB then I think that's good, but if it's over 200 then maybe we should bring it down a little to ensure the reports don't get rejected by the API?

Reply:
The stacktrace from the new mazerunner scenario weighs in at about 30kb, and the payload as a whole at about 40kb. Note: that's pretty-printed JSON collected from mazerunner, so in reality we can expect those sizes to be about 25% smaller.